docs/getting-started/getting-started-with-gossip-ring.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Getting Started with Gossiped Ring (experimental)"
+linkTitle: "Gossip Ring (experimental)"
+weight: 2
+slug: getting-started-with-gossiped-ring
+---
+
+Cortex requires Key-Value store to store the ring. It can use traditional KV stores like Consul or Etcd,
+but now it can also build its own KV store on top of memberlist library using a gossip algorithm.
```

Comment:
I would remove any temporal reference (ie. "now") from the doc, given it's likely this doc will survive a long time.

```suggestion
but it can also build its own KV store on top of memberlist library using a gossip algorithm.
```